src/Documenter.jl

```diff
@@ -8,6 +8,9 @@ Two functions are exported from this module for public use:
 - [`makedocs`](@ref). Generates documentation from docstrings and templated markdown files.
 - [`deploydocs`](@ref). Deploys generated documentation from *Travis-CI* to *GitHub Pages*.
 
+Additionally it provides the unexported [`generate`](@ref), which can be used to
```

Comment:
Perhaps ``[`Documenter.generate`](@ref)`` to make it more obvious that it's unexported?